tests/baselines/reference/callChainWithSuper(target=es2023).js

```diff
@@ -0,0 +1,18 @@
+//// [callChainWithSuper.ts]

```

Comment:
Is this test relevant to your changes?

Reply:
I believe it was created because the `es2023` target was added (filename is `callChainWithSuper(target=es2023).js`). This file [was added](https://github.com/microsoft/TypeScript/pull/46291/files#diff-6bd62e5549898b8187cd7407f28ad8cfc5bd547fac86fb30213002aeaea8c9b3) for the `es2022` target as well.